src/lambda_function/s3_to_glue/app.py

```diff
@@ -31,38 +73,56 @@ def submit_s3_to_json_workflow(messages: list[dict[str,str]], workflow_name):
     glue_client.put_workflow_run_properties(
         Name=workflow_name,
         RunId=workflow_run["RunId"],
-        RunProperties={"messages": json.dumps(messages)},
+        RunProperties={"messages": json.dumps(objects_info)},
     )
 
 
-def lambda_handler(event, context):
-    """
-    The Lambda entrypoint
-
-    Given a list of files in s3 bucket to process, submit the keys and bucket
-    of the new files to a S3 to JSON Glue workflow.
+def lambda_handler(event, context) -> None:
+    """This main lambda function will be triggered by a
+        SQS event and will poll the SQS queue for
+        all available S3 event messages
 
     Args:
-        event (dict): A fileview object
-        context (obj) : An object that provides methods and properties that
-        provides info about the invocation, function, and execution environment
-
-    Returns:
-        (None) Submits new file records to a Glue workflow.
+        event (json): SQS event
+        context: NA
```

Comment:
What is context?

Reply:
https://docs.aws.amazon.com/lambda/latest/dg/python-context.html
We don't use it but it has to be present in any lambda_handler function. I can add a quick note about it in the function.